### DISCUSSION TITLE
exact repository search

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62334] [Repositories]
### Select Topic Area

Bug

### Body

I want to search for PAGER repos and not PAGE.

I tried these and it finds both:

~~~
pager
"pager"
~~~

I tried this

~~~
pager NOT PAGE 
~~~

but it also excludes repos that contain pager only

**Comment by look** [comment]
You could try a regular expression search such as [`/(\s+|^)pager(\s+|$)/`](https://github.com/search?q=%2F%28%5Cs%2B%7C%5E%29pager%28%5Cs%2B%7C%24%29%2F&type=code).

**Reply by ghost** [reply]
its not a code search, its a repository search, as stated in the title

**Reply by look** [reply]
Apologies. I'll move this to the Repositories category.
